code/modules/jobs/job_types/internal_affairs_agent.dm

```diff
@@ -0,0 +1,62 @@
+/datum/job/iaa
+	title = "Internal Affairs Agent"
+	flag = IAA
+	department_head = list("Captain")
+	department_flag = CIVILIAN
+	faction = "Station"
+	total_positions = 2
+	spawn_positions = 2
+	supervisors = "Central Command"
+	selection_color = "#ddddff"
+	chat_color = "#50C878"
+	exp_requirements = 840
+	exp_type = EXP_TYPE_CREW
+
+	outfit = /datum/outfit/job/iaa
+
+	access = list(ACCESS_IAA, ACCESS_COURT, ACCESS_SEC_DOORS, ACCESS_SEC_RECORDS,
+				ACCESS_MEDICAL, ACCESS_CONSTRUCTION, ACCESS_RESEARCH, ACCESS_CARGO)
+	minimal_access = list(ACCESS_IAA, ACCESS_COURT, ACCESS_SEC_DOORS, ACCESS_SEC_RECORDS,
+				ACCESS_MEDICAL, ACCESS_CONSTRUCTION, ACCESS_RESEARCH, ACCESS_CARGO)
+	paycheck = PAYCHECK_MEDIUM
+	paycheck_department = ACCOUNT_CIV
+	mind_traits = list(TRAIT_LAW_ENFORCEMENT_METABOLISM)
+
+	display_order = JOB_DISPLAY_ORDER_IAA
+	departments = DEPARTMENT_COMMAND | DEPARTMENT_SERVICE
+	rpg_title = "Diplomat"
+
+	species_outfits = list(
+		SPECIES_PLASMAMAN = /datum/outfit/plasmaman/lawyer
+	)
+
+/datum/outfit/job/iaa
+	name = "Internal Affairs Agent"
+	jobtype = /datum/job/iaa
+
+	id = /obj/item/card/id/job/iaa
+	belt = /obj/item/pda/iaa
+	ears = /obj/item/radio/headset/headset_iaa
+	uniform = /obj/item/clothing/under/suit/black
+	suit = /obj/item/clothing/suit/toggle/lawyer/black
+	shoes = /obj/item/clothing/shoes/laceup
+	l_hand = /obj/item/storage/briefcase/lawyer
+	l_pocket = /obj/item/laser_pointer
+	r_pocket = /obj/item/assembly/flash/handheld
+	neck = /obj/item/clothing/neck/tie/black
+	glasses = /obj/item/clothing/glasses/sunglasses/advanced
+
+	chameleon_extras = /obj/item/stamp/law
+
+	implants = list(/obj/item/implant/mindshield)
```

Comment:
😠

Reply:
why not? I don't think brainwashing them then order them to send badly forged information about heads to CC isn't a good idea even if it makes sense ICly.

Reply:
Pretty much what @EvilDragonfiend said. They're corporate affairs agents sent by CC themselves, why _shouldn't_ they have a mindshield?

Reply:
It's funny because the Inspectors CC send don't have a mindshield (happened to me one time, i arrested a LEGIT centcom inspector because i thought it was a traitor on disguise) but yeah, it makes sense that something like a CAA would be mindshielded and having the inspector without mindshield has some RP potential for tots/lings/nukies

Reply:
Inspectors don't have mindshields, the only roles that do, round start are security. Let's keep it that way.

Reply:
I am still questionable for not giving mindshield because some shit HoS would not hear them which would be the most important part for CAA.

Reply:
If this isn't mindshielded it's almost like HoP pre-Command-antag-ban, just the ultimate powergame job with access everywhere and thus easy antagging.